cradlepoint/pages/api/editTestPlan.js

```diff
@@ -0,0 +1,32 @@
+const { ObjectId } = require('mongodb');
+import connectToDb from "../../util/mongodb";
+import {testPlanSchema} from "../../schemas/testPlanSchema";
+/*
+  Edits the requested TestPlan from the database
+*/
+
+export default async (req, res) => {
+  if (req.method !== 'PUT') {
+    res.status(405).send({ message: 'Only PUT requests allowed' })
+  }
+  try{
+    const data = req.body;
+    const valid = await testPlanSchema.isValid(data);
+    if (valid){
+      const result = testPlanSchema.cast(data);
+      // Set ID strings to Mongo ObjectId's
+      const newId = ObjectId(result._id);
```

Comment:
Can we not make this newId, and instead leave it as id, since we aren't actually making a new one? Otherwise LGTM